### ISSUE TITLE
Save recording locally

### ISSUE THREAD

**Issue by tejaskdubal** [cats-oss/VideoCast-Swift#50]
First of all, Thanks for this awesome library.

Question/Problem - 
Is it possible to save video locally in a device irrespective of the network state?

Please go through the following cases - 

1. User is not able to connect the device with the network.
2. Streaming is interrupted by the network/server issue.
3. Streaming is interrupted by network/server issues and not able to re-stream at all.

This could be a solution - 
Provide option that allows users to save the recording locally with a specific name.

**Comment by thmatuza** [comment]
Yes, if you uncomment this, you can save video locally in a device.
https://github.com/cats-oss/VideoCast-Swift/blob/0.1.5/Source/Api/iOS/VCSimpleSessionImpl.swift#L232-L244

It will pass an output of the encoder to both the network and storage.

I was thinking about that case, too.
Above code works ok though, my ideal is like this.
1. Encoding h264 and streaming it via RTMP with relatively low video quality
2. Encoding h265 and saving it locally with high video quality

I will probably work on it sometime this year.

**Comment by tejaskdubal** [comment]
Ok, thanks for the immediate response.
